odata-protocol/11.4 Data Modification.md

```diff
@@ -1040,59 +1040,96 @@ through the `ContentIDSupported` property of the
 [`Capabilities.DeepUpdateSupport`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Capabilities.V1.md#DeepUpdateSupportType)
 term, both defined in [OData-VocCap](#ODataVocCap).
 
+For each entity being updated or removed, clients MAY specify an [ETag](#UseofETagsforAvoidingUpdateConflicts) value obtained from a previous request. 
```

Comment:
We are a bit vague in Protocol regarding "ETag values specified in the request body" or how exactly an "entity representation in the request body includes an ETag".

Can we explicitly state, for example in section "[Use of ETags for Avoiding Update Conflicts](https://oasis-tcs.github.io/odata-specs/odata-protocol/odata-protocol.html#UseofETagsforAvoidingUpdateConflicts)", that 

> The representation of Etags within entity representations is format-specific, see for example JSON Format, section [Control Information: etag (odata.etag)](https://docs.oasis-open.org/odata/odata-json-format/v4.02/csd01/odata-json-format-v4.02-csd01.html#ControlInformationetagodataetag).

Reply:
* Created #1971